File: cp_rust/src/bin/abc126-a-changing-a-character.rs
// Created by Ayush Biswas at 2026/06/08 17:01
// https://atcoder.jp/contests/abc126/tasks/abc126_a
use cp_lib::*;

// @code begin
use cpio::*;

sol! {
    fn solution(
        [n, k]: [usize; 2],
        (mut s): [char]
    ) -> String {
        s[k - 1] = s[k - 1].to_lowercase().to_string().chars().next().unwrap();
        s.into_iter().collect()
    }
}
// @code end

File: cp_nim/src/bin/abc126-a-changing-a-character.nim
# Created by Ayush Biswas at 2026/07/18 12:42
# https://atcoder.jp/contests/abc126/tasks/abc126_a
import ../lib/cpio

# @code begin

import strutils

var r = newReader()

var
  (n, k) = r[int, int]
  s = r[string]

s[k - 1] = s[k - 1].toLowerAscii
echo s

# @code end
